### ISSUE TITLE
Downstream release of the odo cli

### ISSUE THREAD

**Issue by girishramnani** [openshift/odo#3852]

**Comment by prietyc123** [comment]
@girishramnani @mohammedzee1000 Can you please provide the detailed description.

**Comment by mohammedzee1000** [comment]
As this is downstream, i cannot give out too much

But basically, we are almost done with this

There is a <redacted> ticket up in the release pipeline which is waiting on QE approval (docs is already done). Once this is done, we can push it to redhat cdn.

Documentation is being updated accordingly in internal release repo.

**Comment by mohammedzee1000** [comment]
The ticket is now ready for push to redhat cdns, pinged and waiting internal <redacted> team member to ship it

**Comment by mohammedzee1000** [comment]
the ticket is now shipped live on redhat cdn
/close
